go-controller/pkg/ovn/policy.go

```diff
@@ -222,33 +223,67 @@ func deleteACLPortGroup(portGroupName, direction, priority, match, action string
 	return nil
 }
 
-func addToPortGroup(ovnNBClient goovn.Client, portGroupName string, portInfo *lpInfo) error {
-	cmd, err := ovnNBClient.PortGroupAddPort(portGroupName, portInfo.uuid)
-	if err == nil {
-		if err = ovnNBClient.Execute(cmd); err != nil {
-			return fmt.Errorf("execute error adding port %s to port group %s (%v)", portInfo.name, portGroupName, err)
+func addToPortGroup(ovnNBClient goovn.Client, portGroupName string, ports ...*lpInfo) error {
+	cmds := make([]*goovn.OvnCommand, 0, len(ports))
```

Comment:
Does the size of this need to be limited? I know the services controller ran into an issue where the command batch was too large for nbdb to process

Reply:
that was cause we are using bash nbctl calls there, since this is using go-bindings it should be more robust

Reply:
and im not sure if there is any size limit to number of transactions in an ovsdb call...should find out